src/api/propensity-api.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Why?

Reply:
Looks like its just my add-on to visual studio code that's complaining.  Removed.